### ISSUE TITLE
Error message decoding strings is wrong or wrongly applied

### ISSUE THREAD

**Issue by KariOlson-PG** [InteractiveAdvertisingBureau/Consent-String-SDK-Java#45]
When decoding Base64 strings, I see this error pretty frequently:
"Expected consent string to contain at least 17bytes, but found only 17 bytes"

This message is either being thrown in error: 
You needed at least x and you got exactly x.  This is not a problem. 

Or the error message is being generated incorrectly and one of the two values is wrong. The correct error should be either of :
"Expected consent string to contain at least 17bytes, but found only 16 bytes"
"Expected consent string to contain at least 18bytes, but found only 17 bytes"

Here are some examples of strings that are generate this error when querying for purpose 5:
BOP02S_OkoBERAKABCENBDA
BOmOjKtOmOjKtAKABBENBDA

**Comment by lanusau** [comment]
Yes, its a misleading message, but the error is only in the message, the actual check is correct and these strings are invalid. The first number is the index of the byte array is being accessed and its counted from 0, so for index 17 you need 18 bytes. 
I will fix it in the next release.
